stages/org.osbuild.systemd.unit.create.meta.json

```diff
@@ -33,6 +33,19 @@
     "    - 'Where' - string",
     "    - 'Type' - string",
     "    - 'Options' - string",
+    "  - 'Socket' section",
+    "    - 'ListenStream' - string",
+    "    - 'ListenDatagram' - string",
+    "    - 'ListenSequentialPacket' - string",
+    "    - 'ListenFifo' - string",
```

Comment:
This should be `ListenFIFO`.
Wasn't sure if case matters for these so I ran a quick check with `systemd-analyze verify` which returned:
```
Unknown key 'ListenFifo' in section [Socket], ignoring.
```

Reply:
Pushed a fix